config/routes.rb

```diff
@@ -15,9 +15,10 @@
   resources :newsletters, only: [:create]
   get  "first_edition" => "home#first_edition_projects"
   get "sponsors" => "home#sponsors"
-  get  "mentee_dashboard" => "mentee_profiles#dashboard"
+  get "mentee_dashboard" => "mentee_profiles#dashboard"
+  resources :mentee_profiles, only: [:show]
+  post "mentee_dashboard/missing_mentor" => "mentee_profiles#missing_mentor"
```

Comment:
please add this route under mentee_profiles resource - take a look at dashboard for mentor profile :)